ReactWindows/ReactNative.Shared/UIManager/AccessibleCanvas.cs

```diff
@@ -0,0 +1,30 @@
+#if WINDOWS_UWP
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Automation.Peers;
+#else
+using System.Windows.Controls;
+using System.Windows.Automation.Peers;
+#endif
+
+namespace ReactNative.UIManager
+{
+    /// <summary>
+    /// Canvas with ReactXP/ReactNative accessibility properties support. 
+    /// </summary>
+    public class AccessibleCanvas : Canvas, IAccessible
+    {
+        /// <inheritdoc />
+        protected override AutomationPeer OnCreateAutomationPeer()
+        {
+            return new AccessibleAutomationPeer<AccessibleCanvas>(this);
+        }
+
+        // TODO: implement runtime change raising event to screen reader #1228350
+        /// <inheritdoc />
+        public AccessibilityTrait[] AccessibilityTraits { get; set; }
+
+        // TODO: implement runtime change raising event to screen reader #1228350
+        /// <inheritdoc />
+        public ImportantForAccessibility ImportantForAccessibility { get; set; }
```

Comment:
>ImportantForAccessibility [](start = 41, length = 25)

What's the default value?

Reply:
Yes, 0, and 0 is explicitly Auto in the enum:
```csharp
public enum ImportantForAccessibility
    {
        /// <summary>
        /// Platform decides which views are important for accessibility and brings
        /// the screen reader focus on those views
        /// </summary>
        Auto = 0,
```

Reply:
I will also add explicit initialization to an enum member to avoid confusion.

---
In reply to: [158562699](https://github.com/reseul/react-native-windows/pull/1#discussion_r158562699) [](ancestors = 158562699)